src/Core/src/Handlers/Editor/EditorHandler.iOS.cs

```diff
@@ -61,16 +61,16 @@ public override Size GetDesiredSize(double widthConstraint, double heightConstra
 				// get an exception; it doesn't know what do to with it. So instead we'll size
 				// it to fit its current contents and use those values to replace infinite constraints
 
-				PlatformView.SizeToFit();
+				var sizeThatFits = PlatformView.SizeThatFits(new CGSize(widthConstraint, heightConstraint));
```

Comment:
Exactly the same logic is applied in the SearchBar, happens the same problem? does it require the same changes?

https://github.com/dotnet/maui/blob/489657c5458691f3a4c360d84b3a53a03b61b03f/src/Core/src/Handlers/SearchBar/SearchBarHandler.iOS.cs#L43

If need the same changes, could you add a test covering the fix in the SearchBar?